cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -74,6 +74,7 @@ func GetEtcdPodSpec(cfg *kubeadmapi.InitConfiguration) v1.Pod {
 			cfg, kubeadmconstants.Etcd, 2379, cfg.CertificatesDir,
 			kubeadmconstants.EtcdCACertName, kubeadmconstants.EtcdHealthcheckClientCertName, kubeadmconstants.EtcdHealthcheckClientKeyName,
 		),
+		Resources: staticpodutil.ComponentResources("100m"),
```

Comment:
in k8s on GCE we have two etcd events:
- the main one with 200m
- the events one with 100m